requirements-macos-torch.txt

```diff
@@ -0,0 +1 @@
+torch==2.2.0
```

Comment:
For some reason, `poetry install` is no longer working on MacOS (apple silicon). But I added this, so torch can be installed manually this way via `pip install -r requirements-macos-torch.txt`